packages/core/src/cosmjs.ts

```diff
@@ -0,0 +1,61 @@
+// File to export the cosmjs structs used in this package
```

Comment:
Why do we need this now?

Reply:
To allow people that use `desmjs` to use cosmjs structs without adding a dependency to `cosmjs/x` inside the `package.json`

Reply:
Can we then replace all the manual exports with something along the lines of `export * from 'cosmjs'`?

Reply:
Didn't know that i can do `export { ... } from 'package'`.
Unfortunally I can't export everyting from a module since we will risk to have export collisions and expose also classes that are related to cosmjs like `SigningCosmWasmClient` that should not be exposed since not usefull.
Btw I have updated this file to use the export statement that you suggest and where was safe to use the `export *`.